src/DebugSession.h

```diff
@@ -35,6 +38,7 @@ class DebugSession : public QObject
 	SymbolTable symTable;
 	QString fileName;
 	bool modified;
+	bool debugUpdates;
```

Comment:
Maybe this variable is not needed, see above. But if you want to keep it ...

I think it's safer to initialize this variable to false:
```bool debugUpdates = false;```
And then only set it to true when the `openmsx_update enable debug` succeeds.

This way you eliminate the (small) window where this state is not yet initialized.